From 363f2d99c9a9f49cabbef2818edb4df61a3f86cf Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Sat, 29 Feb 2020 15:13:02 +0000
Subject: [PATCH 46/73] WIP: dt-bindings: arm: amlogic: add support for the
 Tanix TX5 Max

The Oranth (Tanix) TX5 Max is based on the Amlogic U200 reference
board with an S905X2 chip.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 Documentation/devicetree/bindings/arm/amlogic.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/amlogic.yaml b/Documentation/devicetree/bindings/arm/amlogic.yaml
index ea4f016082c4..ba74854cd6b1 100644
--- a/Documentation/devicetree/bindings/arm/amlogic.yaml
+++ b/Documentation/devicetree/bindings/arm/amlogic.yaml
@@ -145,6 +145,7 @@ properties:
           - enum:
               - amediatech,x96-max
               - amlogic,u200
+              - oranth,tx5-max
               - radxa,zero
               - seirobotics,sei510
           - const: amlogic,g12a
-- 
2.17.1

